src/main/config/secor.common.properties

```diff
@@ -139,6 +139,14 @@ kafka.seed.broker.port=9092
 # to as the chroot.
 kafka.zookeeper.path=/
 
+# Store offset in zookeeper and kafka consumer topic.
+# Possible values: true or false
+kafka.dual.commit.enabled=true
+
+# Storage offset.
+# Possible values: "zookeeper" to read offset from zookeeper or "kafka" to read offset from kafka consumer topic
+kafka.offsets.storage=zookeeper
+
```

Comment:
Maybe we should also add 'offset.storage.topic' property for people to specify the topic name to store the offset.